packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -515,6 +516,20 @@ export class SDKClient {
         throw predefined.HBAR_RATE_LIMIT_EXCEEDED;
       }
       throw sdkClientError;
+    } finally {
+      /**
+       *  For transactions of type CONTRACT_CREATE, if the contract's bytecode (calldata) exceeds 5120 bytes, HFS is employed to temporarily store the bytecode on the network.
+       *  After transaction execution, whether successful or not, any entity associated with the 'fileId' should be removed from the Hedera network.
+       */
+      if (transaction['_callDataFileId']) {
```

Comment:
If for some reason the sdk changes this property name at some point, we would have to change again. Ideal for me would be to keep track of the transacion type in the relay and not rely on specific implementation somewhere else. But this might never happen, so this is also good 👍